documentation/wiki/Mac-Debugging.md

```diff
@@ -0,0 +1,38 @@
+#Debugging with MacOS
+Say unit tests are failing from Microsoft.Build.Engine.UnitTests_netcoreapp2.1_x64
```

Comment:
Can you include some console output from `build.sh -test` showing how to know that this is the thing that's failing?